src/kixi/datastore/schemastore/conformers.clj

```diff
@@ -0,0 +1,16 @@
+(ns kixi.datastore.schemastore.conformers
+  (:require [clojure.core :exclude [integer?]]
+            [clojure.spec :as s]))
+
+(defn -integer?
```

Comment:
I've done x-integer? for this for things that are or can be converted to integers. Shall we start using that as a convention (or define one we will use everywhere)